src/components/CreateForm.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react'
+import styled from 'styled-components/macro'
+import { NavLink, useHistory } from 'react-router-dom'
+
+export default function CreateForm({ onSubmit }) {
```

Comment:
This is your Form component. You could just name it like that. I would remove create here.

Reply:
You are right. Form is probably easier to understand and I will also only have one Form component in my app. Will change it.